plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -36,6 +36,22 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/scheduler/schedulercache"
 )
 
+// predicatePrecomputations: Helper types/variables...
+
+type PredicateMetadataModifier func(pm *predicateMetadata)
+
+var predicatePrecomputations map[string]PredicateMetadataModifier = make(map[string]PredicateMetadataModifier)
+
+var predicatePrecomputeRegisterLock sync.Mutex
```

Comment:
nit: can you please move it above predicatePrecomputations? For me it's always easier to see the lock before the variable it's locking.